echopype/calibrate/calibrate_base.py

```diff
@@ -194,7 +194,15 @@ def _apply(self, echodata) -> Dict[str, xr.DataArray]:
         #         }
         #     )
 
-        return {var: env_params[var] for var in ("temperature", "salinity", "pressure")}
+        if self.data_kind == "stationary":
+            # renaming time2 to ping_time is necessary because we are performing
+            # calculations with the beam groups that use ping_time
```

Comment:
This was intended to refer to time3. Also, I think it can be confusing to keep track of whether `time#` refers to a dimension from the Platform or Environment group, when you're operating on both; I added a clarification
```suggestion
            # renaming time3 (from Platform group) to ping_time is necessary because 
            # we are performing calculations with the beam groups that use ping_time
```